src/Fantomas.Core/Context.fs

```diff
@@ -540,10 +545,11 @@ let ifElse b (f1: Context -> Context) f2 (ctx: Context) = if b then f1 ctx else
 
 let ifElseCtx cond (f1: Context -> Context) f2 (ctx: Context) = if cond ctx then f1 ctx else f2 ctx
 
-let ifStroustrupElse = ifElseCtx (fun ctx -> ctx.Config.ExperimentalStroustrupStyle)
+let ifStroustrupElse =
+    ifElseCtx (fun ctx -> ctx.Config.MultilineBracketStyle = ExperimentalStroustrup)
```

Comment:
Use `ctx.Config.ExperimentalStroustrupStyle` in this case. I like the additional member you created.